research-hub-web/src/app/components/home/research-activity/research-activity.component.html

```diff
@@ -29,7 +29,7 @@ <h3>{{ item.name }}</h3>
             </div>
 
             <!-- Panel Image -->
-            <div fxHide.lt-lg class="research-category-img">
+            <div fxHide.lt-md class="research-category-img">
                 <img src="{{ item.image?.url }}" alt="{{ item.name }}-image">
```

Comment:
Not specific to this PR. But remove the `-image` from `alt` text. Screen readers already identify images, see:
- https://blog.hubspot.com/marketing/image-alt-text
- https://moz.com/learn/seo/alt-text#:~:text=Because%20search%20engines%20can't,in%20your%20alt%20text.